src/InstrumentationEngine/ProfilerManager.cpp

```diff
@@ -587,7 +624,17 @@ HRESULT CProfilerManager::AddInstrumentationMethod(
         return E_OUTOFMEMORY;
     }
 
-    return this->AddInstrumentationMethod(pInstrumentationMethod.release(), ppInstrumentationMethod);
+    CComPtr<CConfigurationSource> pSource;
+    pSource.Attach(new CConfigurationSource(bstrModulePath));
+    if (nullptr == pSource)
+    {
+        return E_OUTOFMEMORY;
+    }
```

Comment:
Have we set the flag to not throw on "new" for this codebase? If not, there is no point in testing this. Othewise, if you want to do this, then you should use "new nothrow" above.